source/includes/_tipser-elements-analytics.md

```diff
@@ -1,189 +1,308 @@
-
-_In the screenshot above the user has clicked "add to cart" button which resulted in emitting "Add to cart" event._
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?
----------        | -----------    | -----------        
-`E-commerce`     | `add`          | Yes (with one product attached) 
-
-## Remove from cart
-
-This event is emitted when product is removed from cart in checkout dialog.
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?
----------        | -----------    | -----------       
-`E-commerce`     | `remove`       | Yes (with one product attached)
-
-## Checkout
-
-This event is emitted when checkout dialog is opened with a product (or several products).
-
-[![](product_checkout.png)](/images/product_checkout.png)
-
-_In the screenshot above checkout dialog has been displayed and "Checkout" event has been emitted for a product "Retexturising SERUM"_
-
-### Additonal notes ###
-
-Please note that "checkout" is not the same thing as "purchase". The event is emitted even if the product checkout is never finalized.
-The event is emitted every time checkout dialog is opened. This means that there may be several "Checkout" events emitted for product that was purchased just 
-once or that was never purchased.
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?     
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
-
-
-## Purchase
-
-This event is emitted when a product (or several products) is purchased.
-
-[![](purchase_event.png)](/images/purchase_event.png)
-
-_In the screenshot above the user has finalized the payment and the order confirmation dialog has been displayed. "Purchase" event has been emitted._
-
-### GA event format ###
-
-Category         | Action         | Standard Enhanced Ecommerce event?        
----------        | -----------    | -----------               
-`E-commerce`     | `checkout`     | Yes (with one or many products attached)
+# Tipser Elements Analytics
+
+Tipser has implemented an event for the Tipser Elements Analytics that users of the script can listen to in order to gather analytics about Tipser interactions and feed them into their own analytics engines.
+
+## General overview of the Tipser Elements Analytics collector
+By listening to the event `tw-track` certain Tipser events can be gathered and pushed into an external analytics engine.
+
+The event contains the object detail that contains the following items:
+
+`detail` object
+
+```yaml
+{
+  action: string, # what action is taken on the target, mandatory
+  target: string, # where this action is taken, mandatory
+  description: string, # describing what is being logged
+  object: # details about the tracked object, e.g. collection ID, product ID, etc.
```

Comment:
Maybe it's good to mention some obvious properties here, like product name or product price?

Reply:
How do you see it ? I've removed the structure of object from here, as it can be very different depending on user interaction (it may be not product related), and focused on providing consistent structure for every interaction.

Reply:
Overall the description is ok. I'm only referring to your comment here:
```

Reply:
But I don't insist. It's just a friendly suggestion, not a change request.

Reply:
Can you give precise an example ? Otherwise I'm not really sure what you want to add here.

> I think it would be nice to give some more obvious examples

Obvious examples are given below. This is a schema.

Reply:
So for example the comment could be:
`# details about the tracked object, e.g. product name, product price, etc.`

Reply:
```suggestion
  object: # details about the tracked object, e.g. product name, product price, etc.
```